### PR TITLE
Add localisation, english as default

### PR DESCRIPTION
**Pull request by vschaeidt**

### CODE REVIEW

File: termine-fe/src/App.js [hunk 1]
```diff
@@ -156,8 +157,10 @@ function App() {
           state: INFOBOX_STATES.ERROR,
           msg:
             error.response.status === 410
-              ? "Leider ist der Termin inzwischen nicht mehr buchbar, bitte einen anderen Termin wählen."
-              : "Ein unbekannter Fehler ist aufgetreten, bitte Seite neu laden.",
+              ? i18n._(
+                  t`The appointment is no longer available, please select another free appointment.`
+                )
+              : i18n._(t`Un unknown error occurred, please reload the page.`),
```
Comment (globin) [inline termine-fe/src/App.js:163]:
```suggestion
              : i18n._(t`An unknown error occurred, please reload the page.`),
```

File: termine-fe/src/App.js [hunk 2]
```diff
@@ -185,8 +188,10 @@ function App() {
...
```
Comment (globin) [inline termine-fe/src/App.js:194]:
```suggestion
              : i18n._(t`An unknown error occurred, please reload the page.`),
```

File: termine-fe/src/App.js [hunk 3]
```diff
@@ -229,13 +234,13 @@ function App() {
             {window.config.longInstanceName}
           </h3>
           <Link className="button" to={TAB.BOOK}>
-            Termine buchen
+            <Trans>Book appointments</Trans>
           </Link>
           <Link className="button" to={TAB.BOOKED}>
-            Meine Buchungen
+            <Trans>My Appointments</Trans>
```
Comment (globin) [inline termine-fe/src/App.js:240]: Inconsistent  capitalisation

File: termine-fe/src/Components/Booking.js [hunk 4]
```diff
@@ -45,9 +49,11 @@ export default function Booking({
       onFocus={() => onFocusHandler()}
     >
       <fieldset className="input-group vertical">
-        <legend>Terminbuchung</legend>
+        <legend>
+          <Trans>Booking</Trans>
+        </legend>
         <label htmlFor="firstName" className="displayFlex">
-          Vorname {errors.firstName && renderInputRequired()}
+          <Trans>Name</Trans> {errors.firstName && renderInputRequired()}
```
Comment (globin) [inline termine-fe/src/Components/Booking.js:56]:
```suggestion
          <Trans>Given Name</Trans> {errors.firstName && renderInputRequired()}
```

File: termine-fe/src/Components/InfoBox.js [hunk 5]
```diff
@@ -22,15 +25,18 @@ export const InfoBox = ({
...
+                {formatTime(
+                  bookedAppointment.startDateTime,
+                  bookedAppointment.lengthMin
+                )}{" "}
+                has been booked
+              </Trans>
             </h3>
             <p className="doc">
-              Teilen Sie dem Patienten jetzt den Berechtigungscode mit:
+              <Trans>Notify the patient of the Access-code:</Trans>
```
Comment (globin) [inline termine-fe/src/Components/InfoBox.js:39]:
```suggestion
              <Trans>Notify the patient of their access code:</Trans>
```

File: termine-fe/src/Views/BookingHistoryView.js [hunk 6]
```diff
@@ -57,12 +63,24 @@ export default function BookingHistoryView({
...
+              <Trans>Date of Appointment</Trans>
+            </th>
+            <th>
+              <Trans>Booked at</Trans>
+            </th>
+            <th>
+              <Trans>Surname</Trans>
+            </th>
+            <th>
+              <Trans>Name</Trans>
```
Comment (globin) [inline termine-fe/src/Views/BookingHistoryView.js:76]:
```suggestion
              <Trans>Given Name</Trans>
```

File: termine-fe/src/Views/BookingHistoryView.js [hunk 7]
```diff
@@ -57,12 +63,24 @@ export default function BookingHistoryView({
...
+            </th>
+            <th>
+              <Trans>Mobile No.</Trans>
+            </th>
+            <th>
+              <Trans>Access-code</Trans>
```
Comment (globin) [inline termine-fe/src/Views/BookingHistoryView.js:82]:
```suggestion
              <Trans>Access Code</Trans>
```

File: termine-fe/src/App.js (hunk 3 above)
Comment (vschaeidt) [reply termine-fe/src/App.js]: removed capitalisation of appointments, good catch!

**Pull request merged**